docker-compose.yaml

```diff
@@ -1,16 +1,18 @@
-version: "3"
```

Comment:
General question, not really part of your change (maybe).
we used to write mac-ipxe files directly on our main netboot server. And since every container on the netboot server ran as root, there was no problem in reading the files in the mounted folder realtime.

But now, we don't run every container as root anymore. Does that mean, that the described behavior will also change?